modules/exploits/windows/antivirus/ams_xfr.rb

```diff
@@ -50,16 +50,13 @@ def initialize(info = {})
   end
 
   def windows_stager
-
-    exe_fname = rand_text_alphanumeric(4+rand(4)) + ".exe"
-
     print_status("Sending request to #{datastore['RHOST']}:#{datastore['RPORT']}")
-    execute_cmdstager({ :temp => '.', :cgifname => exe_fname })
```

Comment:
`cgifname` does nothing. It is copy pasta from a couple of other Windows modules developed around the same time and use a similar code pattern. It has been removed.